compose-builder/add-device-usb-camera.yml

```diff
@@ -21,7 +21,6 @@ services:
     image: ${DEVICE_SVC_REPOSITORY}/device-usb-camera${ARCH}:${DEVICE_USBCAM_VERSION}
     ports:
       - "127.0.0.1:59983:59983"
-      - "127.0.0.1:8554:8554/tcp"
```

Comment:
Isn't this needed to access the embedded RTSP service? Removing access to it makes this a breaking chnage.

Reply:
Hmm if we leave this in, then we would need to have a way to remove it when rtsp-server is enabled otherwise it will create a port conflict. More thoughts on the design may be needed.

Reply:
Maybe a variable that is set to this in Makefile by default and then set to nothing if new `rtsp-server` option is used. Tricky to do the whole line, so maybe just the port???